drivers/include/slipdev.h

```diff
@@ -120,13 +128,19 @@ typedef struct {
 typedef struct {
     netdev_t netdev;                        /**< parent class */
     slipdev_params_t config;                /**< configuration parameters */
-    chunk_ringbuf_t rb;                     /**< Ringbuffer to store received frames.       */
+    chunk_ringbuf_t rb;                     /**< Ringbuffer to store received NET frames.   */
                                             /* Written to from interrupts (with irq_disable */
                                             /* to prevent any simultaneous writes),         */
                                             /* consumed exclusively in the network stack's  */
                                             /* loop at _isr.                                */
 
     uint8_t rxmem[CONFIG_SLIPDEV_BUFSIZE];  /**< memory used by RX buffer */
+
+#if IS_USED(MODULE_SLIPDEV_STDIO)
+    chunk_ringbuf_t rb_config;              /**< Ringbuffer to store received CONFIG frames.*/
+    uint8_t rxmem_config[CONFIG_SLIPDEV_BUFSIZE]; /**< memory used by RX buffer */
+    kernel_pid_t coap_server_pid;
```

Comment:
Please don't foce `slipdev_stdio` to require a CoAP server, make this a separate (pseudo)module.